lib/file_transfer_agent/encrypt_util.js

```diff
@@ -121,8 +121,7 @@ function encrypt_util(encrypt, filestream, temp)
       infile.on('close', function ()
       {
         outfile.write(cipher.final());
-        outfile.close();
-        resolve();
```

Comment:
Don't we need resolve() anymore?

Reply:
why the change is not showing in the PR?

Reply:
the line just below this message, line 124